controller/src/conf/application.conf

```diff
@@ -27,6 +27,21 @@ config {
     #Unique name for this cluster.
     cluster = ${CLUSTER_NAME}
 
+    #Flag to enable authorization
+    authorizationEnabled = ${AUTHORIZATION_ENABLED}
+
+    #Users and passwords settings for default Pravega authorization
+    userPasswordFile = ${USER_PASSWORD_FILE}
+
+    #TLS settings
+    tlsEnabled = ${TLS_ENABLED}
```

Comment:
Is it correct that this setting and corresponding env variable are no longer used?

Reply:
This is the server side setting. The changes are for the client side. The server will still need to be explicitly told that it needs to enable TLS.

Reply:
ok